requirements.txt

```diff
@@ -1,62 +1,56 @@
 Django==2.1.8
-Pillow==4.3.0                   # PIL fork (Python Imaging Library)
-djangorestframework==3.7.7      # API v1
+Pillow==6.0.0                   # PIL fork (Python Imaging Library)
+djangorestframework==3.9.4      # API v1
 django-filter==2.1.0           # Filtering for DRF
 # python-memcached==1.57          # Memcache. Used by Mailinglists to fetch from Sympa.
 # Upstream is missing Python 3 patches
 git+https://github.com/JelteF/python-memcached@patch-1#egg=python-memcached==1.58
 markdown2==2.3.7                # textarea text formatting
 pytz                            # Timezone lib
-stripe==2.27.0                  # Stripe payment
-icalendar==4.0.1                # iCalendar generation
-google-api-python-client==1.6.5
+stripe==2.28.1                  # Stripe payment
```

Comment:
I'm a bit unsure about this upgrade.
I originally built the new payments API with `2.27.0`, and everything was working great.
Tested it with `2.28.0` before setting up the PR, but random tests were failing because of API problems every time.

Maybe they fixed it with `2.28.1`, but I think it would be wise to look into.

Reply:
The tests are failing locally, but that has to do with me not having a test-stripe-token, and they are working on CI, so it should be fine